src/main/scala/org/broadinstitute/dsde/workbench/leonardo/Boot.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.stream.ActorMaterializer
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.LazyLogging
+import net.ceedubs.ficus.Ficus._
+import org.broadinstitute.dsde.workbench.leonardo.api.LeoRoutes
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object Boot extends App with LazyLogging {
+
+  private def startup(): Unit = {
+
+    val config = ConfigFactory.load()
+
+    // we need an ActorSystem to host our application in
+    implicit val system = ActorSystem("leonardo")
+    implicit val materializer = ActorMaterializer()
+    import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
I guess this is needed for the call to `recover` below. I'm wondering if we should `import system.dispatcher` in Leo instead of using the global execution context (and manage the ECs in the Akka config).